bin/docker/build

```diff
@@ -7,9 +7,33 @@ IFS=$'\n\t'
 
 read -r version < ".ruby-version"
 
+DOCKER_TAG="byos_sinatra:latest"
+APP_ENV="production"
+BUILD_ARGS=("--build-arg" "RUBY_VERSION=${version}")
+while [ -n "${1-}" ]; do
```

Comment:
:art: Would it make sense to make this a case statement? This would have the added benefit of making the code more self-describing since you could see that using `-t` would tag the project and `--dev` would be for dev only.

Would it also make sense to make `--dev` the default option. I ask because, at least for me, I'm generally building for local development purposes rather than for production to quickly spot check things.